.circleci/config.yml

```diff
@@ -37,11 +43,35 @@ jobs:
             - vendor
             - resources
             - app
+      - run:
+          name: Create implementation code by extending BaseIndex
+          command: |
+            cat <<'PHPSRC' > /var/www/mysite/www/app/src/MySolrIndex.php
+            <?php
+              use Firesphere\SolrSearch\Indexes\BaseIndex;
+              use SilverStripe\CMS\Model\SiteTree;
+              class MySolrIndex extends BaseIndex {
+              public function init() {
+                $this->addClass(SiteTree::class);
+                $this->addFulltextField('Title');
+                $this->addFulltextField('Content');
+              }
+              public function getIndexName() {
+                return 'MySolr';
```

Comment:
The index name should be something like `TestIndex`, so it's clear that if it ever shows up in a real environment, it's not an actually used index.

`MySolr` is a bit ambigouoaeuaeuoaus